lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -10,9 +10,13 @@ const ReactDom = require('react-dom');
 const Modal = require('dosomething-modal');
 
 function getLocalStorageKey(user, campaign) {
+  // @TODO: probably need a better way to get the run ID consistently.
+  const languageCode = (Object.keys(campaign.campaign_runs.current)[0]);
```

Comment:
Well, the Onboarding isn't happening for non-US, so yeah. I know still not ideal, but it's some duct-tape in the meantime.